simple_generator_doc.py

```diff
@@ -0,0 +1,12 @@
+
+
+"""It is description of our module"""
+
```

Comment:
Remove kebab
(this file)

Reply:
is-it-kebab?
and_it_is_snake?
should i remove all file? ( which i did for half an hour:( )